sycl/doc/SharedLibraries.md

```diff
@@ -269,6 +269,22 @@ in the `Name` field of the property.
 List of exported symbols is represented in the same way, except the
 corresponding set has the name `SYCL/exported symbols`.
 
+#### Function pointers support
+
+In order to support co-existence of device function pointers feature together
+with dynamic linking, after device code split process `sycl-post-link`
+collects uses of pointers to functions and definitions of functions with
+`indirectly_callable` attribute and attaches info about them to corresponding
+device images via device image properties mechanism.
```

Comment:
Which flow should be used when device code wasn't split?
The image's properties will say that there's a need for functions of this and that signatures but there's no other device binary image to fulfill the need.

Reply:
The property doesn't say about need. It says whether function with such signature was used or not. In case something is missing, it is not possible to understand in SYCL runtime or compiler, but I'd expect some build error emitted by underlying backend.